apps/mux/src/config.tsx

```diff
@@ -31,7 +31,7 @@ import {
   ContentType,
   CompatibleFields,
   SelectedFields,
-} from 'shared-dam-app/src/utils';
+} from '@contentful/dam-app-base/src/utils';
```

Comment:
Yes. I didn't look at this at all, yet. Same for the sku picker.

Reply:
My 2cts: since this code that is in both the [SKU](https://github.com/contentful/apps/blob/0dc16ee5775ec2645231233d3024a7aea898f275/lib/shared-sku-app/src/AppConfig/fields.ts#L48-L81) and the [DAM](https://github.com/contentful/apps/blob/0dc16ee5775ec2645231233d3024a7aea898f275/lib/shared-dam-app/src/AppConfig/fields.ts#L38-L71) base) is generically useful (that's why the Mux app is using it despite not being build on top of the DAM base) it'd be great to pull that into its own library. This way it'll be more discoverable for folks building apps that are neither DAM nor SKU apps.

Reply:
@realityking 

Yes, there is quite some overlapping code in both packages. E.g. also the AppConfig and the logic around it is nearly identical.

For now, I would publish the packages as they are and maybe follow up with a more generic package that contains utility methods/classes

Reply:
@shikaan I changed it so that all types and util methods are exported from `index.tsx`. That way everything can be imported via `@contentful/(dam|sku)-app-base`.